packages/terra-docs/src/Docs.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Docs.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content to be displayed as the name
+   */
+  name: PropTypes.string,
+};
+
+const defaultProps = {
+  name: 'default',
+};
+
+const Docs = ({ name, ...customProps }) => {
+  const DocsClassNames = cx([
+    'docs',
+    customProps.className,
```

Comment:
I can log this against the generator, but I wanted to point out the className here is bound. The generator might need a few updates to handle this if we want to be more cognizant of it when generating components

Reply:
I have a generator PR i could make some minor updates to

Reply:
https://github.com/cerner/generator-terra-module/pull/147

Reply:
I logged an issue and commented on the PR incase we decide to handle them separately:

https://github.com/cerner/generator-terra-module/issues/148